community-operators/node-problem-detector/node-problem-detector.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,171 @@
+  - type: SingleNamespace
+    supported: true
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: false
+  install:
+    spec:
+      clusterPermissions:
+      - serviceAccountName: node-problem-detector-operator
+        rules:
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          # the operator needs to be able to bind the cluster role
+          # system:node-problem-detector to the node-problem-detector service account
+          - clusterrolebindings
+          verbs:
+          - "*"
+        - apiGroups:
+          - security.openshift.io
+          resources:
+          # the operator needs to be able to add the node-problem-detector service account
+          # to the list of accounts that can use the privileged SCC
+          - securitycontextconstraints
+          verbs:
+          - "*"
+        # The following sections for nodes and nodes/status are
+        # copied from the system:node-problem-detector clusterrolebinding
+        - apiGroups:
+          - ""
+          resources:
+          - nodes
+          verbs:
+          - get
+        - apiGroups:
+          - ""
+          resources:
+          - nodes/status
+          verbs:
+          - patch
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+          - update
+      permissions:
+      - serviceAccountName: node-problem-detector-operator
+        rules:
+        - apiGroups:
+          - node-problem-detector.operator.k8s.io
+          resources:
+          - "*"
+          verbs:
+          - "*"
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - events
+          - configmaps
+          - secrets
+          - services
+          - endpoints
+          - serviceaccounts
+          verbs:
+          - "*"
+        - apiGroups:
+          - apps
+          resources:
+          - daemonsets
+          verbs:
+          - "*"
+      deployments:
+      - name: node-problem-detector-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: node-problem-detector-operator
+          template:
+            metadata:
+              labels:
+                name: node-problem-detector-operator
+            spec:
+              containers:
+                - name: node-problem-detector-operator
+                  image: registry.svc.ci.openshift.org/openshift/origin-v4.0:node-problem-detector-operator
+                  ports:
+                  - containerPort: 60000
+                    name: metrics
+                  command:
+                  - node-problem-detector-operator
+                  imagePullPolicy: Always
+                  env:
+                    - name: WATCH_NAMESPACE
+                      valueFrom:
+                        fieldRef:
+                          fieldPath: metadata.namespace
```

Comment:
`metadata.annotations['olm-targetNamespaces']`

Reply:
This is a single-namespace singleton operator. The operator and the operand are supposed to run in the same ns. What's the correct `fieldPath` in that setup?

Reply:
With that setup, it is fine to leave the `fieldPath` as is.